src/main/java/io/wisetime/connector/jira/JiraDao.java

```diff
@@ -211,7 +211,7 @@ private void upsertWorklogSeqId(final long seqId) {
 
   Optional<Long> getWorklogSeqId() {
     try {
-      final long seqId = query().select("SELECT seq_id FROM sequence_value_item WHERE seq_name='Worklog'")
+      final long seqId = query().select("SELECT seq_id FROM SEQUENCE_VALUE_ITEM WHERE seq_name='Worklog'")
```

Comment:
Found out that table name in MySQL is case sensitive in most Unix varieties.

Reply:
https://stackoverflow.com/questions/6134006/are-table-names-in-mysql-case-sensitive